PWGDQ/Core/VarManager.h

```diff
@@ -1253,14 +1256,44 @@ void VarManager::FillPair(T1 const& t1, T2 const& t2, float* values)
     values[kPsiPair] = (t1.sign() > 0) ? TMath::ASin((v1.Theta() - v2.Theta()) / xipair) : TMath::ASin((v2.Theta() - v1.Theta()) / xipair);
   }
 
-  // CosTheta Helicity calculation
+  // set as default values
+  // TO DO: get the correct values from CCDB
+  double CenterOfMassEnergy = 13600.;                                                                       // GeV
```

Comment:
Please add a static data member to the VarManager which will hold the beam energy, instead of hardcoding the value here. You should also add a setter function, so that the user can configure this value. Otherwise this would not work for other colliding energies. In the future, we should use the metadata/CCDB to get this value.